ios/FluentUI/BottomSheet/BottomSheetViewController.swift

```diff
@@ -82,6 +85,34 @@ public class BottomSheetViewController: UIViewController {
 
         updateResizingHandleViewAccessibility()
     }
+    
+    public override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+        if animator?.state == .active {
+            // panning started, don't update the height
+            return
+        }
+        updateHeight()
+    }
+
+// MARK: private helpers
+    
+    private func updateHeight(){
+        // update the view layout for collapsed mode
+        if currentState == .collapse
+        {
+            if let window = self.view.window {
+                view.frame = targetCollapseFrame(with: window)
+                view.layoutIfNeeded()
+            }
+        }
+    }
+
+    private func targetCollapseFrame(with window:UIWindow)->CGRect
+    {
+        let windowFrame = window.frame
+        return CGRect(x: 0, y: (windowFrame.height - collapsedHeight), width: windowFrame.width, height: collapsedHeight)
```

Comment:
nit { bracket style. pleae make sure you install swift-lin to catch these type of syntax errors. https://github.com/harrieshin/fluentui-apple#swift-lint